modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/S3.scala

```diff
@@ -42,6 +42,14 @@ object S3 {
 
     def bucketName: String =
       f.split("://").last.split("/").head
+
+    def keyName: String =
```

Comment:
That's folder. It cannot have a key (filename). It just defeats the purpose of the type.